policies/optimized-images.md

```diff
@@ -39,60 +62,70 @@ Optimized image policies introduce a set of restrictions (policies) on images th
 
 On a web page, the number of pixels of a container determines the resolution of an image served inside. It is unnecessary to use an image that is much larger than what the viewing device can actually render; for example, serving a desktop image to mobile contexts, or serving an image intended for high-pixel-density screens to a low-pixel-density device. This results in unnecessary network traffic and downloaded bytes. `oversized-images` is a policy controlled feature that restricts images to be no more than X times bigger than the container size.
 
-When a document is disallowed to use `oversized-images` policy, its `<img>` elements that are more than X times larger than its container size will be rendered as a placeholder image.
-
+When a document disallows the `oversized-images` policy, the `<img>` elements that are more than X times larger than the container size will be replaced with placeholder images.
 
-#### Specification
+To try the `oversized-images` policy, register a token [here](https://developers.chrome.com/origintrials/#/trials/active) and specify the policy via HTTP `Feature-Policy` header (see section above for more details).
 
-- The default oversizing ratio is 2.
 
-    **Note**: The goal is to eventually introduce a syntax for specifying the maxmimum oversizing ratio to be allowed.
+#### Specification
 
-    In practice, they would look something like this:
+- The default allowlist for `oversized-images` is `*(inf)`. This means for pages of all origins,
+all `<img>` elements will be allowed and rendered correctly by default.
 
-    ```html
-    <iframe allow="oversized-images(4)"></iframe>
-    ```
-    That would apply a policy in which the maximum oversizing ratio allowed is set to 4.
+- An `oversized-images` policy can be specified via:
 
-    Feature policies combine in subframes, and the minimum value of the downscaling ratio will be applied, so if that frame embedded another, which the syntax:
+    **1. [HTTP `Feature-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy) response header:**
```

Comment:
You could link to the spec here, too -- https://w3c.github.io/webappsec-feature-policy/#feature-policy-http-header-field